ps3t_game_list.php

```diff
@@ -96,7 +96,14 @@
         
         $nextpage = 1;

```

Comment:
With a dynamic page number check we can remove anything that uses `$nextpage` and `$k` since we won't need to check if the next page exists.